mcsema/BC/Function.cpp

```diff
@@ -166,6 +211,37 @@ static void InlineSubFuncCall(llvm::BasicBlock *block,
   (void) new llvm::StoreInst(call, mem_ptr, block);
 }
 
+// Add a invoke to another function and then update the memory pointer with the
+// result of the function. It also needs to stash the stack and frame pointer which
+// can be restored before handling the exception handler.
+static void InlineSubFuncInvoke(llvm::BasicBlock *block,
+                                llvm::Function *sub, llvm::BasicBlock *ifnormal,
+                                llvm::BasicBlock *ifexception, const NativeFunction *cfg_func) {
+  llvm::IRBuilder <> ir(block);
+  auto get_sp_func = gModule->getFunction("__mcsema_get_sp");
```

Comment:
Let's rename these to `__mcsema_get_stack_pointer` and `__mcsema_get_frame_pointer`